client/rpc-api/Cargo.toml

```diff
@@ -21,6 +21,7 @@ parking_lot = "0.10.0"
 sp-core = { version = "2.0.0-alpha.5", path = "../../primitives/core" }
 sp-version = { version = "2.0.0-alpha.5", path = "../../primitives/version" }
 sp-runtime = { path = "../../primitives/runtime" , version = "2.0.0-alpha.5"}
+sc-chain-spec = { path = "../chain-spec" , version = "2.0.0-alpha.5"}
```

Comment:
`rpc-api` should not depend on `client` stuff. The whole point of this crate is to avoid brining in 90% of substrate when just needing the APIs. Can we move the type to some primitives?